src/components/Layout/TrendingBox.jsx

```diff
@@ -57,6 +57,10 @@ const Box = styled.div`
 		width: 100%;
 		border: 1px solid var(--color-6);
   }
+
+  @media (max-width: 750px) {
```

Comment:
Em cima já tem um display none com 650px. Eu achei que tinha ficado legal. Da uma olhada aí e se for 750 deixe ele em cima também fica mais fácil de visualizar porque os css estão indentados.